assets/src/styles/blocks/Covers/TakeActionCovers.scss

```diff
@@ -154,12 +154,12 @@
       display: block;
 
       --block-covers--card-btn-- {
+        color: white;
         background-color: $orange;
         border-color: $orange;
-      }
 
-      &:hover, &:focus, &:focus-within {
-        --block-covers--card-btn-hover-- {
+        &:hover, &:focus {
+          color: white;
```

Comment:
that's something I didn't realize before, but you have to explicitly duplicate a property so that it can be extended as a variable right ? like `color` is already defined in the parent, but it's here too for this reason ?

Reply:
Yep, it's here because our base theme does give a separate color to background and border. For us white works in both cases, but that's not necessarily the case if you choose another background.